src/main/java/com/arpnetworking/clusteraggregator/configuration/ClusterAggregatorConfiguration.java

```diff
@@ -429,6 +455,8 @@ public Builder setDatabaseConfigurations(final Map<String, DatabaseConfiguration
         private RebalanceConfiguration _rebalanceConfiguration;
         @NotNull
         private String _clusterHostSuffix = "";
+        @NotNull
+        private Props _clusterJoinActor = new NonJoiningClusterJoiner.Builder().build();
```

Comment:
At the end of the day it becomes an actor.  I don't want the people configuring the service to have to know that they're actually building a Props that will get created into an actor.  As far as they know, they want to configure an actor.